docs/src/content/docs/guides/authoring-content.md

```diff
@@ -361,6 +361,31 @@ A code block’s optional title can be set either with a `title="..."` attribute
   ```
   ````
 
+## Details
+
+Details (also known as “disclosures” or “accordion”) are useful to conceal information that is not immediately relevant to the user.
+The user can reveal further information with an interaction to toggle the visibility of the content.
```

Comment:
This sentence was maybe saying the same thing twice a bit: “reveal further information” and “toggle the visibility of the content”.

I think we can maybe boil it down to this?

```suggestion
Users can click a short summary to expand and view the full content.
```